src/main/resources/styles/reporting.css

```diff
@@ -191,43 +163,49 @@ table.stats-table td {
 	background-color: white;
 }
 
-table.stats-table .header {
+table.stats-table tr.header {
 	background-color: #66CCEE;
 }
 
-table.stats-table .summary {
+table.stats-table tr.summary {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (tr.summary) is overqualified, just use .summary without element name.](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/1993444831/issues/source?bid=3059308&fileBranchId=3120225#l170)